kubernetes/salt/config/master.d/salt-master.conf

```diff
@@ -5,11 +5,11 @@ interface: 0.0.0.0
 event_return: mysql
 
 mysql:
-  host: '127.0.0.1'
+  # salt does not support specifying the UNIX socket location here - as a workaround,
```

Comment:
I looked at that - that appears to be the doc for manipulating a mysql instance via a salt script, whereas the pillar does not support that: https://github.com/saltstack/salt/blob/develop/salt/pillar/mysql.py#L102-L112